build.gradle

```diff
@@ -172,20 +194,100 @@ jar {
     }
 }
 
-processResources {
-    // this will ensure that this task is redone when the versions change.
-    inputs.property "version", project.version
-    inputs.property "mcversion", project.minecraft.version
+reobf {}
 
-    // replace stuff in mcmod.info, nothing else
-    from(sourceSets.main.resources.srcDirs) {
-        include 'mcmod.info'
+task devJar(type: Jar) {
+    manifest {
+        attributes 'Built-By': 'Electrical-Age Team'
+        attributes 'Build-Date': new Date().format("yyyy-MM-dd HH:mm:ss")
+        attributes 'Built-JDK': System.getProperty('java.version')
+        attributes 'Implementation-Title': project.name
+        attributes 'Implementation-URL': 'https://github.com/jrddunbr/ElectricalAge'
+        attributes 'Implementation-Vendor': 'Electrical-Age Team'
+        attributes 'Implementation-Vendor-Id': 'net.electricalage.eln'
+        attributes 'Project-Url': 'https://eln.ja13.org/'
+        attributes 'Source-Compatibility': project.sourceCompatibility
+        attributes 'Target-Compatibility': project.targetCompatibility
+    }
 
-        // replace version and mcversion
-        expand 'version': project.version, 'mcversion': project.minecraft.version
+    classifier = 'dev'
+
+    from sourceSets.main.output
+
+    // Copy libs jar files
+    // from { configurations.external.collect { it.isDirectory() ? it : zipTree(it) } }
```

Comment:
Nit: Can probably just delete this, right?

Reply:
As of right now yes, but didn't know if you want to shade some other stuff in later or something.  If you want to shade much more you really should make a shade configuration (again see GT6's build script, or upgrade to RFG's build system).